cmd/admin.go

```diff
@@ -57,8 +58,59 @@ to make automatic initialization process more smoothly`,
 			},
 		},
 	}
+
+	subcmdChangePassword = cli.Command{
+		Name:   "change-password",
```

Comment:
@lunny @appleboy we already have `create-user` so I don't really see the issue with `change-password`. Then it becomes self-explaining :)